showcase/gapic-showcase/src/test/java/com/google/showcase/v1beta1/it/ITGdch.java

```diff
@@ -76,27 +74,29 @@ public class ITGdch {
   private String projectId;
   private URI tokenUri;
 
-  @Before
-  public void setup() throws IOException, URISyntaxException {
+  @BeforeEach
+  public void setup(@TempDir Path tempDir) throws IOException, URISyntaxException {
     transportFactory = new InterceptingMockTokenServerTransportFactory();
-    prepareCredentials();
+    prepareCredentials(tempDir);
     settings =
         EchoSettings.newBuilder()
             .setCredentialsProvider(FixedCredentialsProvider.create(initialCredentials))
             .build();
   }
 
-  @After
+  @AfterEach
   public void tearDown() throws InterruptedException {
     if (client != null) {
       client.close();
       client.awaitTermination(TestClientInitializer.AWAIT_TERMINATION_SECONDS, TimeUnit.SECONDS);
     }
   }
 
-  private void prepareCredentials() throws IOException {
+  private void prepareCredentials(Path tempDir) throws IOException {
     // Copy file so it can be referenced by Path even in native-image builds
-    File caCertFile = tempFolder.newFile(CA_CERT_FILENAME);
+    File caCertFile = new File(tempDir.toFile(), CA_CERT_FILENAME);
+    //    Path caCertPath = tempDir.resolve(CA_CERT_FILENAME);
+    //    File caCertFile = Files.createFile(caCertPath);
```

Comment:
Were these lines left in for testing locally?

Reply:
oops. Thanks for catching this.